pom.xml

```diff
@@ -31,6 +31,7 @@
 
         <spring-boot.version>2.4.2</spring-boot.version>
         <spring-cloud.version>2020.0.1</spring-cloud.version>
+        <chaos-monkey-spring-boot.version>2.3.2</chaos-monkey-spring-boot.version>
```

Comment:
In prevision of next Spring Boot version upgrade, do you know how to handle them the spring boot chaos monkey library?

Reply:
That's what they write on their website (https://codecentric.github.io/chaos-monkey-spring-boot/#docs)

```
  | Chaos Monkey 1.0.1 | Chaos Monkey 1.5.0 | Chaos Monkey 2.0.0 | Chaos Monkey 2.0.1 | Chaos Monkey 2.0.2 | Chaos Monkey 2.1.1 | Chaos Monkey 2.2+
-- | -- | -- | -- | -- | -- | -- | --
Spring Boot 1.5.0 - 1.5.19 | no | yes | no | no | no | no | no
Spring Boot 2.0.0 - 2.0.8 | yes | no | yes | yes | yes | yes | yes
Spring Boot 2.1.0+ | no | no | no | no | yes | yes | yes
```

Reply:
Ok thank you. So there is not one version of Chaos Monkey per version of Spring Boot. It will be easier for upgrades.